src/types.rs

```diff
@@ -75,6 +85,11 @@ pub struct OutputScene {
     pub schedule_time_ms: Option<f64>,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, JsonSchema)]
+pub struct AudioComposition {
```

Comment:
This is user-facing name, AudioComposition does not sound good. Think about a better name, if nothing else maybe AudioMixer, AudioMixerOptions or just Audio would be better.